CONTRIBUTING.md

```diff
@@ -61,6 +61,23 @@ $ ./package.sh   # Create the conjur-env binary in the vendor dir and a ZIP of t
 $ ./test.sh      # Run the test suite
 ```
 
+#### Integration Testing
+
+To run the buildpack integration tests, the test script needs to be given the API endpoint and admin credentials
+for a CloudFoundry installation. These are provided as environment variables to the script:
+
+```sh-session
+$ export CF_API_ENDPOINT=https://api.sys.cloudfoundry.net
+$ CF_ADMIN_PASSWORD=... ./test.sh
```

Comment:
They both could be used either way. My reasoning was that the `API_ENDPOINT` isn't particularly sensitive, and so it's fine if it set in the session's environment.

The `ADMIN_PASSWORD` is more sensitive though, so we only want it set in the environment for `test.sh`.

Reply:
Why is one of these `export`ed but not the other?